### ISSUE TITLE
Create code samples for orders

### ISSUE THREAD

**Issue by VictorAvelar** [VictorAvelar/mollie-api-go#72]
**Is your feature request related to a problem? Please describe.**
As a maintainer, I will like to offer code samples for the most common mollie workflows, this will make the use of the library easier and also reduce the entry barrier for people new to go.

**Describe the solution you'd like**
Replicate orders examples provided on [mollie's documentation](https://docs.mollie.com).

Provide examples for:
- [Create order](https://docs.mollie.com/reference/v2/orders-api/create-order)
- [Get order](https://docs.mollie.com/reference/v2/orders-api/get-order)
- [Cancel order](https://docs.mollie.com/reference/v2/orders-api/cancel-order)
- [Update order](https://docs.mollie.com/reference/v2/orders-api/update-order)
- [List order](https://docs.mollie.com/reference/v2/orders-api/list-orders)
- [Update order line](https://docs.mollie.com/reference/v2/orders-api/update-orderlines)
- [Cancel order line](https://docs.mollie.com/reference/v2/orders-api/cancel-orderlines)
- [Create order line](https://docs.mollie.com/reference/v2/orders-api/create-orderlines)
- [Create order payment](https://docs.mollie.com/reference/v2/orders-api/create-order-payment)
- [Create order refund](https://docs.mollie.com/reference/v2/orders-api/create-order-refund)
- [List order refunds](https://docs.mollie.com/reference/v2/orders-api/list-order-refunds)

Make sure to also add at least one example test case so that godoc can render it.

**Describe alternatives you've considered**
No other alternatives were considered, but I'm open to suggestions.

**Additional context**
N/A

**Comment by RamonSmit** [comment]
Is this still on the map for creating? I'm really having difficulties with implementing orders at this moment. 😅

**Comment by VictorAvelar** [comment]
Hey man, sorry for the delay answering.

TL;DR yes, I will try to allocate some time for it during the weekend.

**Comment by VictorAvelar** [comment]
@RamonSmit I added some basic examples on #145 , if you need further assistance just post more concrete questions here.

**Comment by RamonSmit** [comment]
@VictorAvelar 

This does it for me! Thank you so much!!
